netty/src/main/java/io/grpc/transport/netty/NettyServerStream.java

```diff
@@ -85,8 +93,12 @@ protected void inboundDeliveryPaused() {
 
   @Override
   protected void internalSendHeaders(Metadata.Headers headers) {
-    channel.writeAndFlush(new SendResponseHeadersCommand(id(),
-        Utils.convertServerHeaders(headers), false));
+    SendResponseHeadersCommand headersCommand =
+        new SendResponseHeadersCommand(
+            Utils.shouldFlush(channel, !getMethodType().serverSendsOneMessage()),
```

Comment:
Made comment about flushing behavior for unary calls. The coalescing of flushes after read is doc'd on the Utils.shouldFlush method